tests/test_continuous_delivery_manager.py

```diff
@@ -68,35 +68,14 @@ def test_set_repository_info(self):
 
     @patch("vsts_cd_manager.continuous_delivery_manager.ContinuousDelivery")
     @patch("vsts_cd_manager.continuous_delivery_manager.Account")
-    def test_setup_continuous_delivery___account_doesnt_exist(self, mock_account, mock_cd):
-        # Mock the CD Client
-        mocked_cd = mock_cd.return_value
-        # Mock the Account Client
-        mocked_account = mock_account.return_value
-        mocked_account.create_account.return_value = AccountModel()
-        mocked_account.account_exists.return_value = False
-        # create CD manager
-        cdman = ContinuousDeliveryManager(None)
-        # Mock the vsts info call
-        cdman._get_vsts_info = self._mock_get_vsts_info
-        # set required values
-        cdman.set_azure_web_info('group1', 'web1', 'fakeCreds', 'sub1', 'subname1', 'tenant1', 'South Central US')
-        cdman.set_repository_info('repoUrl1', 'master1', 'token1', None, None)
-        # call setup
-        with self.assertRaises(RuntimeError) as context:
-            cdman.setup_continuous_delivery('staging', 'AspNet', "https://account1.visualstudio.com", False, 'token2', None, None)
-        self.assertTrue('does not exist' in str(context.exception))
-
-    @patch("vsts_cd_manager.continuous_delivery_manager.ContinuousDelivery")
-    @patch("vsts_cd_manager.continuous_delivery_manager.Account")
     def test_setup_continuous_delivery___create_account(self, mock_account, mock_cd):
```

Comment:
do you want to add some negative test cases as well?

Reply:
Both positive and negative test scenarios are already in place.